### DISCUSSION TITLE
Customize U-Net in "unet" Folder - Need Configuration Guide

### DISCUSSION THREAD

**Discussion by Damarcreative** [lllyasviel/Fooocus#2291] [Q&A]
In the "models" folder, there is a special space for adding U-Net models. However, I'm having trouble figuring out the settings required to use U-Net.

Can you provide guidance on how to configure to use a custom U-Net?, parameters, or specific configuration files that require attention would be greatly appreciated.

**Comment by mashb1t** [comment]
moved to discussions, @lllyasviel tagging as assignee. Feel free to answr when you have time.
